### DISCUSSION TITLE
iframes not rendering in posts

### DISCUSSION THREAD

**Discussion by mrashco** [CaiJimmy/hugo-theme-stack#709] [Q&A] [answered]
Hi there,

I am just trying to embed an iframe into a post.md for example. I'm new to Hugo/markdown, but from reading online, markdown supports iframe. But when I paste in an iframe I see `<!-- raw HTML omitted -->` instead, is this because of SCSS pre-processing or something?

Any help would be greatly appreciated, thanks!

**Comment by mrashco** [comment]
Update, I've got it working from a [comment here](https://github.com/matcornic/hugo-theme-learn/issues/462), which is good. It'd still be nice if I could just paste in an iframe though :/

**Comment by CaiJimmy** [comment, marked as answer]
Set `markup.goldmark.renderer.unsafe= true` in your config file.

https://gohugo.io/getting-started/configuration-markup

**Reply by mrashco** [reply]
Thanks for the reply! I'll give it a go!

**Reply by mrashco** [reply]
Worked a treat, ty :) ... great theme btw!
